dropbox-sdk-android/src/main/java/com/dropbox/core/android/AuthActivity.kt

```diff
@@ -119,20 +117,12 @@ public open class AuthActivity : Activity() {
             createStateNonce(getSecurityProvider())
         }
 
-        val queryParams = QueryParamsUtil.createExtraQueryParams(
-            tokenAccessType = mState.mTokenAccessType,
-            scope = mState.mScope,
-            includeGrantedScopes = mState.mIncludeGrantedScopes,
-            pkceManagerCodeChallenge = mState.mPKCEManager.codeChallenge
-        )
-
```

Comment:
This is where the crash started when `mState.mTokenAccessType` was `null`.  It worked fine on OAuth2 PKCE flows, but not standard OAuth2 Flows.

I've updated the sample app to support both types to ensure we are testing both flows regularly.